app/assets/javascripts/components/features/compose/containers/sensitive_guide_container.jsx

```diff
@@ -0,0 +1,40 @@
+import { connect } from 'react-redux';
+import TextIconButton from '../components/text_icon_button';
+import { changeComposeSensitivity } from '../../../actions/compose';
+import { Motion, spring } from 'react-motion';
+import { injectIntl, defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  message: { id: 'compose_form.sensitive_message', defaultMessage: 'Please be sure to mark NSFW if the image you are trying to post is erotic content' }
```

Comment:
I think it might make sense to broaden the language here, maybe something like:

> Please be sure to mark the post as NSFW if the image is not safe for work environments.

(note the period at the end too.)

I think being vague might help instance admins, but maybe this is text they'd like to customize.